docs/data/joy/pages.ts

```diff
@@ -5,16 +5,17 @@ const pages = [
     icon: 'DescriptionIcon',
     children: [
       { pathname: '/joy-ui/getting-started/overview' },
+      { pathname: '/joy-ui/getting-started/installation' },
       { pathname: '/joy-ui/getting-started/usage' },
       { pathname: '/joy-ui/getting-started/tutorial' },
       { pathname: '/joy-ui/getting-started/templates' },
       {
-        pathname: '/joy-ui/core-features',
-        subheader: 'core-features',
+        pathname: '/joy-ui/main-features',
+        subheader: 'main-features',
         children: [
-          { pathname: '/joy-ui/core-features/global-variant' },
-          { pathname: '/joy-ui/core-features/automatic-adjustment' },
-          { pathname: '/joy-ui/core-features/perfect-dark-mode' },
+          { pathname: '/joy-ui/main-features/global-variants' },
```

Comment:
We forgot to add the redirections for the old URLs.

Reply:
Just to make sure－was this fixed on the below-linked PR?

Reply:
Yes, with this line https://github.com/mui/material-ui/pull/33462/files#diff-7a0cbac74402caf0e99ba338a9e7c42684580a1f85e149110c3c2ca6e38055afR377